src/backend/Csrs.Api/Models/ModelExtensions.cs

```diff
@@ -98,19 +125,41 @@ public static async Task<MicrosoftDynamicsCRMssgCsrsfile> ToDynamicsModel(this F
                 SsgIncomeonorder = ToDecimal(file.IncomeOnOrder),
                 SsgPartyenrolled = GetPartyEnrolled(file.PartyEnrolled),
 
+                SsgRecalculationorderedbythecourt = ConvertToBool(file.RecalculationOrderByCourt),
+
                 //SsgSharedparenting = true; ???
                 //SsgSplitparentingarrangement = true; ??
                 //SsgRegistrationdate = ConvertToDTOffset(file.), ??
                 //SsgStyleofcauseapplicant = "Applicant", ??
                 //SsgStyleofcauserespondent = "Respondent", ??
+                    
+            };
+            
+            return  dynamicsFile;
+        }
 
-                SsgBCCourtLevelODataBind = dynamicsClient.GetEntityURI("ssg_csrsfiles", courtlevelId),
-                SsgBCCourtLocationODataBind = dynamicsClient.GetEntityURI("ssg_csrsfiles", courtLocationId),
-
+        public static MicrosoftDynamicsCRMssgCsrsfile ToDynamicsModel(this CSRSAccountFile file)
+        {
+            MicrosoftDynamicsCRMssgCsrsfile dynamicsFile = new MicrosoftDynamicsCRMssgCsrsfile
+            {
+                SsgCsrsfileid = file.FileId,
+                SsgFmepfileactive = ConvertToBool(file.IsFMEPFileActive),
+                SsgFmepfilenumber = file.FMEPFileNumber,
+                SsgSafetyalert = ConvertToBool(file.SafetyAlertRecipient),
+                SsgSafetyconcerndescription = file.RecipientSafetyConcernDescription,
+                SsgSafetyalertpayor = ConvertToBool(file.SafetyAlertPayor),
+                SsgPayorssafetyconcerndescription = file.PayorSafetyConcernDescription,
             };
+
             return dynamicsFile;
         }
 
+
+        public static async Task<Child> ToViewModelAsync(MicrosoftDynamicsCRMssgCsrschild dynamicsChild, IDynamicsClient dynamicsClient, IMemoryCache cache, CancellationToken cancellationToken)
```

Comment:
I thought I removed this function cause it wasn't doing anything and wasnt used.